packages/core/src/api/stageInstances.ts

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable jsdoc/check-param-names */
+
+import type { RequestData, REST } from '@discordjs/rest';
+import {
+	type RESTGetAPIStageInstanceResult,
+	type RESTPatchAPIStageInstanceJSONBody,
+	type RESTPatchAPIStageInstanceResult,
+	type RESTPostAPIStageInstanceJSONBody,
+	type RESTPostAPIStageInstanceResult,
+	Routes,
+} from 'discord-api-types/v10';
+
+export class StageInstancesAPI {
+	public constructor(private readonly rest: REST) {}
+
+	/**
+	 * Creates a new stage instance
+	 *
+	 * @see {@link https://discord.com/developers/docs/resources/stage-instance#get-stage-instance}
+	 * @param body - The data to use when creating a new stage instance
+	 * @param options - The options to use when creating a new stage instance
```

Comment:
```suggestion
	 * @param body - The data for creating the new stage instance
	 * @param options - The options to use when creating the new stage instance
```
Other files phrase it like this (such as `create` and `createChannel` in 
`GuildAPI`), and also this is just better wording overall imo